modules/virt-about-hco-operator.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-architecture.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-hco-operator_{context}"]
+= About hco-operator
+
+The `hco-operator` (HCO) provides a single entry  point for deploying and managing KubeVirt and several helper operators with opinionated defaults, and creates custom resources (CRs) for those operators.
```

Comment:
I think KubeVirt should be OpenShift Virtualization for downstream docs. 

s/KubeVirt/{VirtProductName}

Reply:
@tiraboschi Please see the note here. Should this definition use KubeVirt or OpenShift Virtualization?